db/db.go

```diff
@@ -1079,7 +1027,13 @@ func saveValidators(data *types.EpochData, tx *sqlx.Tx, client rpc.Client) error
 
 	if data.Epoch == 0 {
 		var err error
-		genesisBalances, err = BigtableClient.GetValidatorBalanceHistory([]uint64{}, 0, 0)
+
+		indices := make([]uint64, 0, len(data.Validators))
```

Comment:
code style: I have seen this construct a few times now. This might be is a good fit for a new utily function to reduce boilerplate code

Reply:
should be done in a separate PR